CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+## [master]
+
+[master]: https://github.com/DripEmail/drip-ruby/compare/v2.0.0...HEAD
+
+* Your contribution here!
+
+## [`2.0.0`] (2018-03-27)
+
+[`2.0.0`]: https://github.com/DripEmail/drip-ruby/compare/v1.0.0...v2.0.0
+
+* Purchases endpoint removed and replaced with orders.
+
+## [`1.0.0`] (2017-12-27)
+
+[`1.0.0`]: https://github.com/DripEmail/drip-ruby/compare/v0.0.12...v1.0.0
+
+* Add broadcasts endpoints
```

Comment:
Probably a good idea to add these points to the actual release description as well.

Reply:
At least going forward, certainly. I inferred the backfill from the commits, so I don't feel super confident in it.